docs/moonray/index.md

```diff
@@ -0,0 +1,76 @@
+# {{product}} documentation
+
+{{product}} is a performant, lightweight, secure and
+opinionated distribution of **Kubernetes** which includes everything needed to
+create and manage a scalable cluster suitable for all use cases.
+
+You can find out more about {{product}} on this [overview page] or
+see a more detailed explanation in our [architecture documentation].
+
+![Illustration depicting working on components and clouds][logo]
+
+```{toctree}
+:hidden:
+:titlesonly:
+Home <self>
+tutorial/index
+howto/index
+explanation/index
+reference/index
+```
+
+
+---
+
+## In this documentation
+
+````{grid} 1 1 2 2
+
+```{grid-item-card} [Tutorial](tutorial/index)
+
+**Start here!** A hands-on introduction to {{product}} for new users
+```
+
+```{grid-item-card} [How-to guides](howto/index)
+
+**Step-by-step guides** covering key operations and common tasks
+```
+
+````
+
+````{grid} 1 1 2 2
+
+
+```{grid-item-card} [Reference](reference/index)
+
+**Technical information** - specifications, APIs, architecture
+```
+
+```{grid-item-card} [Explanation](explanation/index)
+
+**Discussion and clarification** of key topics
+```
+
+````
+
+---
+
+## Project and community
+
+{{product}} is an open source project which welcomes community involvement,
+contributions, suggestions, fixes and constructive feedback.
+
+- Our [Code of Conduct]
+- How to [contribute]
+
+
+<!-- IMAGES -->
+
+[logo]: https://assets.ubuntu.com/v1/843c77b6-juju-at-a-glace.svg
+
+<!-- LINKS -->
+
+[Code of Conduct]: https://ubuntu.com/community/ethos/code-of-conduct
+[contribute]: index
+[overview page]: /explanation/about
```

Comment:
This doesn't render. Let's add a TODO here or add the contribution page please.

Reply:
I'm just going to cut the bits we haven't added content for at this point